cloudigrade/config/settings/prod.py

```diff
@@ -1,9 +1,11 @@
 from .base import *  # noqa
 
+DEBUG = env.bool('DJANGO_DEBUG', default=False)
```

Comment:
I believe this should be `False`. When would we want to enable this in prod?

If we do think this could happen, then we should drop the `DEBUG = ...` from each of the environment-specific files and just let it be in base.